FirebaseAuth/Sources/Swift/Auth/Auth.swift

```diff
@@ -1409,43 +1115,42 @@ extension Auth: AuthInterop {
 
   /// Sets `languageCode` to the app's current language.
   @objc open func useAppLanguage() {
-    kAuthGlobalWorkQueue.sync {
-      self.requestConfiguration.languageCode = Locale.preferredLanguages.first
-    }
+    setLanguageCode(Locale.preferredLanguages.first)
   }
 
   /// Configures Firebase Auth to connect to an emulated host instead of the remote backend.
   @objc open func useEmulator(withHost host: String, port: Int) {
```

Comment:
I think this code shouldn't be on a production framework cause it won't happen. Is there any way get rid of it without removing the ability to use it whereas developers are coding or testing on pre-production environments?

Reply:
It could be possible, but I'm not convinced the usability impact would be worth the relatively small size overhead increase.

Reply:
Sure, maybe an approach like surrounding the function with a comment (one above and one to close it) or any kinda  special macro compiler for those who wants to remove it from production could make the trick. Who wants to remove it from prod, could build the frameworks by itself.